### DISCUSSION TITLE
getting "unable to render code block" error for preview page of ipynb(python notebook) file in MSFT Edge browser

### DISCUSSION THREAD

**Discussion by kenichi-segawa** [community/community#64876] [New to GitHub] [answered]
Hi community,

I wanted some help understanding how to fix the following error, "unable to render code block" every time when I try to preview page of ipynb(python notebook) file in Edge browser. I'm highlighting edge because it works in my Chrome browser without any errors, so I know it's not related to the repo/file. Is this related to my browser setting? Any inputs would be appreciated.

Thanks

**Comment by d-suter** [comment, marked as answer]
Hey @kenichi-segawa 

I just listed you some stuff that helped me in the past if I was facing errors when trying to preview `.ipynb` files in Microsoft Edge. 

1. **Clear Cache & Cookies**: Begin by clearing the cache and cookies for Edge. Outdated cache might sometimes cause rendering issues.

2. **Browser Extensions**: Edge extensions can occasionally interfere with content rendering. I suggest disabling your extensions one by one, refreshing the page after each, to see if any particular extension is causing the problem.

3. **Update Microsoft Edge**: Ensure you're running the latest version of Microsoft Edge. Updates can contain fixes for minor glitches and compatibility issues.

4. **Hardware Acceleration**: Navigate to Edge settings, then to `System`. Here, you'll find an option named "Use hardware acceleration when available". Try toggling this option. Some users have found that adjusting this setting can resolve certain display issues.

5. **Version Compatibility**: If you're using the Chromium version of Edge (the one that shares its base with Chrome), the rendering should ideally be similar to Chrome. If you're on an older version of Edge, consider updating.

6. **Alternative Approach**: As an interim solution, you could consider using external platforms or tools like `nbviewer` to view your `.ipynb` files. Just paste the link to your GitHub `.ipynb` file there, and it should render appropriately.

Hope these steps help you resolve the issue.

**Reply by kenichi-segawa** [reply]
Thank you so much! It was the extension causing my preview error. Now it's fixed!!!

**Reply by Acutehu** [reply]
Thanks so much.
The issue is from extension

**Reply by clabordec** [reply]
Thank you so much, I removed three of my extensions, and now I am able to view my code.

**Reply by LauragarciaDS** [reply]
Muchas gracias! fue un problema de extensión también para a mi :D

**Reply by Akshara-03** [reply]
I have no extension in Microsoft edge.In that case, What I want to do? please say

**Reply by eliasinul** [reply]
Great, I turned off all the extensions and spotted the "one that was bugging this .ipynb rendering". Thanks ! Surprisingly it was a youtube related extension!

**Reply by naltmann** [reply]
I turned off my ad blocker and it worked

**Reply by shashank-112** [reply]
for me it's because of "Dark Mode - Night Eye" extension. if its disabled every thing works just fine.

**Reply by AnasKhan809** [reply]
Yes guys, thanks it was youtube related extension. Just remove the extension it will work immediately.

**Reply by TentacioPro** [reply]
Thankyou so much
I had this issue with chrome and sorted it out by trying to disable all extensions one by one
Found it at my second try with ContentShake AI Extension.❤️

**Reply by shriansh8619** [reply]
Really Works

**Reply by XuShenLZ** [reply]
Thanks so much! I was finally able to see the notebook render again after disabling the extension "Ninja Cookie" (which is used to block cookie selection banners. Was able to use other extensions for the same purpose without breaking Github)

**Reply by link5210** [reply]
Yes guys,thanks it was translation related extension.Just remove the extension it will work immediately.

**Reply by Darshit-Garg** [reply]
Thanks a lot!!
It was an extension related to editing PDFs, turned that off, and it started to work

**Reply by vivekjangid17** [reply]
Thanks a lot, makes me happy

**Comment by xyzhaocs** [comment]
Hello @kenichi-segawa , I have encountered the same problem as you, and I also believe it is a extension issue, but I am not sure which extension it is. Can you tell me?

**Reply by starccy** [reply]
Do you have any extensions that can pop up floating windows? FYI, I encountered the same error and I disabled the Saladict, and then it returned to normal.

**Reply by kenichi-segawa** [reply]
For me, it was the "Translate Selected Text with Google" extension but I'd suggest trying it out by what @ceodavee suggested by "disabling your extensions one by one, refreshing the page after each, to see if any particular extension is causing the problem."

**Reply by xyzhaocs** [reply]
Thank you so much! It was the extension Saladict causing my preview error. Now it's fixed!!!

**Reply by rudrakshkarpe** [reply]
Thanks! +1 it was extension problem for me too. I was using a text clipper extension from YT videos which was causing this issue; simply removed it :D

**Reply by Jue-Xu** [reply]
I disabled SmoothScroll extension.

**Reply by kaaviyanA** [reply]
i have no extensions but still unable to open, please help me out

**Reply by lifefeel** [reply]
In my case, I disabled Awesome ScreenShot.

https://chromewebstore.google.com/detail/nlipoenfbbikpbjkfpfillcgkoblgpmj

**Reply by seansio1995** [reply]
For me, the chrome extension "TopCashback" is interfering with the rendering...

**Reply by cocoaaa** [reply]
For me, it was Saladict

**Reply by Zedseayou** [reply]
For me, it was Truffle

**Reply by wsszh** [reply]
> For me, it was Saladict

Thanks for your reminding! Me too...

**Comment by LipstipCherif** [comment]
it was "Selectext: Copy Text from Videos" for my case, once I deactivated the error disappear

**Reply by back2yes** [reply]
for me it was "SmoothScroll", banned it and worked like a charm

**Reply by pritesh2000** [reply]
> it was "Selectext: Copy Text from Videos" for my case, once I deactivated the error disappear

same here

**Reply by d1tected** [reply]
same here

**Comment by codemigs** [comment]
Turn of all extensions, then refresh, then add the extensions again one by one and see which extension might be causing this. For me, it was the my dark mode extension

**Comment by dkkb** [comment]
Disable Tampermonkey script, works for me.

**Comment by jasleen0213** [comment]
how to disable extension. please help

**Comment by ConvolutedDog** [comment]
**Summary**: Disabling browser extensions that interact with webpage text extraction can resolve this problem, such as `火山翻译 - 划词和网页翻译`, `ContentShake AI`, `Ninja Cookie`, `Awesome ScreenShot`, `TopCashback`, `Saladict`, `Truffle`, `Selectext: Copy Text from Videos`, `SmoothScroll`, `dark mode extension` and `Tampermonkey`.

**Comment by subupatayach** [comment]
gmail, and all mail concert my mail id account acctiovets online support
and all world all contrary all bank accounts actiovtes on online 0balance
acctiovets and online international transaction money created my mail 🆔
accounts actiovtes and segunail tower setelight everything online include
support my mail 🆔 accounts on online.

**Comment by mzouink** [comment]
Just fixed the issue. It was [Dark mode for your browser](https://chromewebstore.google.com/detail/dark-mode-for-your-browse/popkohipheagkijpjmllkbnifaokliim?hl=en-US&utm_source=ext_sidebar) on my end.

**Comment by plusls** [comment]
Try this: https://greasyfork.org/zh-CN/scripts/561923-fix-github-ipynb-render-failed
